packages/react-native-bpk-component-panel/src/__snapshots__/withDivider-test.android.js.snap

```diff
@@ -1,287 +1,101 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Android withDivider should render correctly 1`] = `
-<BpkPanel
```

Comment:
It feels odd that all this has been removed, it might be a side effect of something else

Reply:
This looks like it is since we added the `withTheme` wrapper around the `WithDivider` component so it now exports the divider rather than it outputting a panel

Reply:
Yeah @ojcurtis I think you're right. I got round this by using the regular renderer (instead of shallow), so now they look more like the did before 👍